common/changes/@uifabric/example-app-base/jagore-improve-spec-readability_2018-05-03-21-07.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/example-app-base",
+      "comment": "Modify documentation columns for improved readability.",
+      "type": "patch"
```

Comment:
The type here should be none as this change has no impact on the API surface of our library.

Reply:
Check the Travis CI error:

https://travis-ci.org/OfficeDev/office-ui-fabric-react/builds/374491936?utm_source=github_status&utm_medium=notification